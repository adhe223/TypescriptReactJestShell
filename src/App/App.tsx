import React from 'react';

const App: React.StatelessComponent = () => {
  return (
    <h1>Hi!</h1>
  );
};

export default App;
