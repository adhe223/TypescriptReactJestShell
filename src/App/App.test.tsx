import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import App from './App';

describe('App', () => {
  let wrapper: ShallowWrapper<{}, {}>;

  describe('render applications', () => {
    beforeEach(() => {
      wrapper = shallow(<App />);
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
