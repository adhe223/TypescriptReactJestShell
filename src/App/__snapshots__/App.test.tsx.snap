// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render applications renders correctly 1`] = `
<h1>
  Hi!
</h1>
`;
